Makefile

```diff
@@ -97,8 +97,8 @@ azurefile-windows:
 	CGO_ENABLED=0 GOOS=windows go build -a -ldflags ${LDFLAGS} -o _output/azurefileplugin.exe ./pkg/azurefileplugin
 
 .PHONY: container	
-container: azurefile	
-	docker build --no-cache -t $(IMAGE_TAG) -f ./pkg/azurefileplugin/Dockerfile .
+container:	
+	docker build --no-cache --build-arg LDFLAGS=$(LDFLAGS) -t $(IMAGE_TAG) -f ./pkg/azurefileplugin/Dockerfile .

```

Comment:
build args needs to be passed into `az acr build` which has some trouble parsing

Reply:
otherwise, it's only setting the default `latest` version